confidential-identity/cli/README.md

```diff
@@ -1,110 +1,77 @@
 # Confidential Identity Library
 
-## Simple Claim Prover
-
-CDD Claims and Scope Claims are types of claims that can be attached to an investor's identity on the PolyMesh.
-CDD claims are registered by a trusted CDD provider using [cdd-register-did][cdd-register-did] API, and Scope
-claims are created and added to the chain by the investor using [add_claim][add-claim] API.
-
-CDD claims are generated by a PolyMesh trusted CDD provider during the CDD process (see
-[cdd-register-did][cdd-register-did]), and privately link an investor's unique id (`investor_unique_id`, which
-is kept secret between the investor and CDD provider) to their on-chain identity (`investor_did`, the serialized
-[IdentityId][IdentityId] of the investor). In order to achieve this, the CDD provider has to generate a CDD Id,
-include it in their claim, and submit it to the PolyMesh chain.
-Note that every time an investor requests a new identity from a CDD provider, the same unique identity (which is
-obtained from the Polymath Unique Identity System (PUIS)) must be used to create the CDD claim, even though the
-investor's DID will be different. A CDD claim can be inputted to the `polymath-scp` CommandLine Interface (CLI)
-in JSON format. Let's assume it is saved as `my_cdd_claim.json`:
-```
```

Comment:
what was the reason for removing these?

Reply:
I moved it to the readme of `confidential-identity` (one directory up) so that it is shared between both the CLI and WASM. But if it is confusing, I can duplicate in inside the WASM readme instead.

Reply:
Ok, makes sense. I'm good as long as it's around.